src/utils.py

```diff
@@ -1184,15 +1184,22 @@ def update_config(args: Dict[str, Any], default_seed: int = 123) -> None:
     for k in args:
         if k not in allowed_args:
             raise ValueError(f"Unrecognized arg: {k}")
+    for k in ("env", "approach", "seed"):
+        if not args.get(k, None) and hasattr(CFG, k):
+            # For env, approach, and seed, if we don't pass in a value
```

Comment:
only want to do this for env/approach/seed bc for instance if we set some boolean flag to False, and then in a later test don't pass anything in, we don't want it to retain False value

Reply:
isn't `experiment_id` another one of this kind?

Reply:
fixed